### DISCUSSION TITLE
Sort topics by number of comments

### DISCUSSION THREAD

**Discussion by ghost** [community/community#16617] [Discussions] [answered]
Hi all!

#### 1. Feature name
Sort topics by number of comments

#### 2. Feature description
I would like to know topics were most commented on in the Discussions

#### 3. Concept
![image](https://user-images.githubusercontent.com/95204402/168925383-f0f50d84-f6f9-45d2-8a0b-49ff34d2d306.png)

#### 4. Reason
I'd like to filter posts in order of comments, for example posts with more comments or less comments - sometimes a post doesn't get voted on, but it's something everyone wants

#### 5. Notes
1. I would like to help GitHub and open source communities
2. I would like to know if this idea is good or not, if this idea is valid or not
3. There are several discussions about having a feature to sort the discussions either by number of votes, by amount of time etc. - So... I  requested to sort discussions by number of comments

#### 6. References
- https://github.com/github/feedback/discussions/12657
- https://github.com/github/feedback/discussions/10463
- https://github.com/github/feedback/discussions/10585
- https://github.com/github/feedback/discussions/3328
- https://github.com/github/feedback/discussions/6541
- https://github.com/github/feedback/discussions/3235
- https://github.com/github/feedback/discussions/2867

**Comment by Alexarchangel07** [comment]
Good

**Reply by ghost** [reply]
@Alexarchangel07 thank you for feedback ;D

**Comment by khendri** [comment]
![Uploading Screenshot_2022-05-17-08-24-53-83_439a3fec0400f8974d35eed09a31f914.jpg…]()

**Comment by ghost** [comment, marked as answer]
Duplicate of https://github.com/github/feedback/discussions/10463

**Comment by ghost** [comment]
Good idea.
